docs/src/apireference.md

```diff
@@ -481,17 +480,219 @@ Utilities.@model
 
 ## Bridges
 
-Bridges can be used for automatic reformulation of a certain constraint type into equivalent constraints.
+Bridges can be used for automatic reformulation of constrained variables or
```

Comment:
The distinction between "constrained variables" vs "constraints" isn't very clear. I think this should be defined first with examples. Is a constrained variable a SingleVariable or VectorOfVariables in a set, and a constraint a ScalarAffineFunction or VectorAffineFunction in a set?

Reply:
The distinction was added in a separate PR, see http://www.juliaopt.org/MathOptInterface.jl/dev/apireference/#Variables-1

Reply:
Oh boy. OK I think it's worth linking to that description here (and other places where constrained variables are discussed). The special meaning of "constrained variables" is not obvious from the name.

Reply:
Maybe mention in parens `[`add_constrained_variable`](@ref)/[`add_constrained_variables`](@ref)`